packages/docusaurus/src/client/exports/ComponentCreator.tsx

```diff
@@ -23,7 +23,16 @@ export default function ComponentCreator(
   if (path === '*') {
     return Loadable({
       loading: Loading,
-      loader: () => import('@theme/NotFound'),
+      loader: async () => {
+        const NotFound = (await import('@theme/NotFound')).default;
+        return (props) => (
+          // Is there a better API for this?
```

Comment:
🤪 good catch

There's also a native "Loading" screen but afaik it's never displayed so 🤷‍♂️

Reply:
Yeah, the "native" name isn't the best, but I don't think this will be targeted anyways...